GUI/QtQuick/qml/Overview/BaseBalanceLabel.qml

```diff
@@ -0,0 +1,52 @@
+/*
+
+***********************************************************************************
+* Copyright (C) 2018 - 2020, BlockSettle AB
+* Distributed under the GNU Affero General Public License (AGPL v3)
+* See LICENSE or http://www.gnu.org/licenses/agpl.html
+*
+**********************************************************************************
+
+*/
+import QtQuick 2.15
+
+Rectangle {
+    id: control
+
+    property string label_text: "Confirmed balance"
+    property color label_text_color: "#7A88B0"
```

Comment:
We have a list of palette colors in some common module, please use it from there. Ask @dvajdual for details.